gulpfile.js

```diff
@@ -30,6 +30,7 @@ gulp.task('coverage', function() {
     child_process.execSync('nyc' +
         ' --all' +
         ' --report-dir coverage/' +
+        ' -x "index.js"' +
```

Comment:
We shouldn't ignore `index.js` (no need to add coverage for it right now, but we should eventually).